src/QA.Contribution.Test.Journey/Page/ContactUs.cs

```diff
@@ -14,32 +15,52 @@ public ContactUs(ScenarioContext context) : base(context)
         private readonly string _messageLocator = "//*[@id='description']";
         private readonly string _submitMessageLocator = "//*[@id='submitContact']";
         private readonly string _subjectLocator = "//*[@id='subject']";
-        private readonly string _errorLocator = "//*[@class='alert alert-danger']";
-        private readonly string _successLocator = "//*[text()='as soon as possible.']";
         private readonly string _nameLocator = "//*[@id='name']";
         private readonly string _phoneLocator = "//*[@id='phone']";
+        private readonly string _successMessageHeaderLocator = "//*[contains(text(),'Thanks for getting in touch')]";
+        private readonly string _successSecondHeaderLocator = "//*[text()='We'll get back to you about']";
+        private readonly string _successLastLocator = "//*[text()='as soon as possible.']";
+        private readonly string _errorLocator = "//*[@class='alert alert-danger']";
 
         public string ClickSend()
         {
             Driver.GetClickableElement(By.XPath(_submitMessageLocator)).Click();
             return Driver.Url;
         }
 
-        public string EnterEmailAddress()
+        public string GetErrorMessage()
         {
-            var email = Faker.Internet.Email();
-            var emailField = Driver.GetClickableElement(By.XPath(_emailLocator));
-            emailField.Clear();
-            emailField.SendKeys(email);
-            return email;
+            var errorAlert = Driver.GetClickableElement(By.XPath(_errorLocator));
+            return errorAlert.Text;
+        }
+
+        public string GetSuccessMessageHeader()
+        {
+            var successAlert = Driver.GetClickableElement(By.XPath(_successMessageHeaderLocator));
+            return successAlert.Text;
+        }
+
+        public string GetSuccessSecondHeaderLocator()
+        {
+            var successAlert = Driver.GetClickableElement(By.XPath(_successSecondHeaderLocator));
+            return successAlert.Text;
+        }
+
+        public string GetSuccessLastLocator()
+        {
+            var successAlert = Driver.GetClickableElement(By.XPath(_successLastLocator));
+            return successAlert.Text;
         }
 
-        public string EnterInvalidEmailAddress()
```

Comment:
removed unused method and passed the details from feature file to give more understanding